inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -125,12 +126,87 @@ Eigen::Isometry2d InLaneCruisingPlugin::compute_heading_frame(const lanelet::Bas
   return carma_wm::geometry::build2dEigenTransform(p1, yaw);
 }
 
+std::pair<double, size_t> min_with_exclusions(const std::vector<double>& values, const std::unordered_set<size_t>& excluded) {
```

Comment:
This method needs comments either here or make it private and add the comments in the header.

Reply:
made it private